.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
-  name: nodejs-16
```

Comment:
@spadgett Is this file actually used in CI? Based on [configuration](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/console-plugin-template/openshift-console-plugin-template-main.yaml)
```
build_root:
  image_stream_tag:
    name: release
    namespace: openshift
    tag: console-plugin-test-cypress
```
It doesn't have `from_repository: true` so I guess `.ci-operator.yaml` is not used in CI? 
```
build_root:
  from_repository: true
```

Reply:
I'm not sure if this is actually being used. Here is the log where I see registry.access.redhat.com/ubi9/nodejs-18:latest

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_console-plugin-template/77/pull-ci-openshift-console-plugin-template-main-images/1894694425653678080/artifacts/build-logs/console-plugin-template-amd64.log

@cajieh Do you know?

Reply:
Let's open a bug as a follow up since that problem is not new if that works for you @yapei. Thanks for catching this